src/types/gallery/index.ts

```diff
@@ -25,5 +25,6 @@ export type GalleryContextType = {
     syncWithRemote: (force?: boolean, silent?: boolean) => Promise<void>;
     setNotificationAttributes: (attributes: NotificationAttributes) => void;
     setBlockingLoad: (value: boolean) => void;
+    setDropZoneActive: (value: boolean) => void;
```

Comment:
where are we using it?

Reply:
Yes, it isn't necessary now will remove it.